website/src/css/custom.css

```diff
@@ -0,0 +1,9 @@
+:root {
```

Comment:
Add the copyright header (see lint warning on D20065823).

Reply:
Done.